src/test/scala/firrtlTests/formal/VerificationSpec.scala

```diff
@@ -0,0 +1,55 @@
+package firrtlTests.formal
+
+import firrtl.{SystemVerilogCompiler}
+import firrtl.testutils.FirrtlFlatSpec
+import logger.{LogLevel, Logger}
+
+class VerificationSpec extends FirrtlFlatSpec {
+  behavior of "Formal"
+
+  it should "generate Verilog statements within FORMAL block" in {
```

Comment:
This text is outdated now. Thanks for making the changes to the emission :)

Reply:
Fixed in 22876578ddfc1246f9341279f78b242397b79a06